src/main/java/com/synopsys/integration/blackduck/service/BlackDuckService.java

```diff
@@ -58,6 +58,7 @@
     public static final BlackDuckPath BOMIMPORT_PATH = new BlackDuckPath("/api/bom-import");
     public static final BlackDuckPath SCANSUMMARIES_PATH = new BlackDuckPath("/api/scan-summaries");
     public static final BlackDuckPath UPLOADS_PATH = new BlackDuckPath("/api/uploads");
+    public static final BlackDuckPath SCAN_DATA_PATH = new BlackDuckPath("/api/scan/data");
```

Comment:
This api would need to have / at the end. 
eg: /api/scan/data/

Reply:
I don't believe that is accurate, this is the correct way for the BlackDuckPath to be constructed.

Reply:
So I just tested and without the slash I get a 404 status code and with the slash I get a 201 status code. Not sure why this endpoint is different.

Reply:
That sounds like a bug somewhere.

Reply:
The API inside Blackduck is /scan/data/ that's the reason why I mentioned that.